File: problems/C/e006.c
/* Find the difference between the sum of the squares of the first one hundred
   natural numbers and the square of the sum */

#include <stdio.h>
#include <math.h>

int main(void)
{
    int squareSum = 0;
    int sumSquare = 0;

    for (int i = 1; i <= 100; i++)
    {
        squareSum += i;
        sumSquare += pow(i, 2);
    }

    squareSum = pow(squareSum, 2);
    printf("%i\n", squareSum - sumSquare);
    return 0;
}

File: problems/Python/e006.py
''' Find the difference between the sum of the squares of the first one hundred
natural numbers and the square of the sum '''

def main():
    ''' Driver function '''
    sum_square = sum([x**2 for x in range(1, 101)])
    square_sum = sum([x for x in range(1, 101)])**2
    print(square_sum - sum_square)

if __name__ == "__main__":
    main()
